### PR TITLE
release: v3.6.0 release changes (PROJQUAY-1861)

### PR DESCRIPTION
**Pull request by thomasmckay**
Downstream build changes for release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### v3.6.0
```
Comment (thomasmckay) [inline CHANGELOG.md:1]: This file, and the first entry, determines the version displayed in quay UI

File: config.py [hunk 2]
```diff
@@ -452,9 +452,9 @@ def create_transaction(db):
         }
     else:
         BRANDING = {
-            "logo": "/static/img/quay-horizontal-color.svg",
-            "footer_img": None,
-            "footer_url": None,
+            "logo": "/static/img/RH_Logo_Quay_Black_UX-horizontal.svg",
+            "footer_img": "/static/img/RedHat.svg",
+            "footer_url": "https://access.redhat.com/documentation/en-us/red_hat_quay/3/",
         }
```
Comment (thomasmckay) [inline config.py:458]: sets the default images

File: config.py [hunk 3]
```diff
@@ -760,7 +760,7 @@ def create_transaction(db):
     if os.environ.get("RED_HAT_QUAY", False):
         DOCUMENTATION_ROOT = "https://access.redhat.com/documentation/en-us/red_hat_quay/3/"
     else:
-        DOCUMENTATION_ROOT = "https://docs.projectquay.io/"
+        DOCUMENTATION_ROOT = "https://access.redhat.com/documentation/en-us/red_hat_quay/3/"
```
Comment (thomasmckay) [inline config.py:763]: set the link that docs show

**Review by harishsurf — APPROVED** [review]
lgtm

**Review by harishsurf — APPROVED** [review]
lgtm

**Pull request merged**